src/executables/loadjava.cpp

```diff
@@ -64,13 +70,14 @@ parse_argument (int argc, char *argv[])
   struct option loadjava_option[] =
   {
     {"overwrite", 0, 0, 'y'},
+    {"jni", 0, 0, 'j'},
     {0, 0, 0, 0}
   };
 
   while (1)
     {
       int option_index = 0;
-      int option_key = getopt_long (argc, argv, "yh", loadjava_option, &option_index);
+      int option_key = getopt_long (argc, argv, "y:jh", loadjava_option, &option_index);
```

Comment:
Why did you add ':'?
Using ':' means there are arguments.
Does the '-y' option have additional arguments?